hack/lib/common-var-init.sh

```diff
@@ -78,6 +78,7 @@ EVICTION_PRESSURE_TRANSITION_PERIOD=${EVICTION_PRESSURE_TRANSITION_PERIOD:-"1m"}
 
 # ensures all places of cert/node naming related to hostname are lower-cased for consistency
 lohostname=$(hostname | tr '[:upper:]' '[:lower:]')
+hostip=$(hostname -i)
```

Comment:
Not sure if hostname -i works on MacBook.

Reply:
I don't think Arktos-up supports MacBook

Reply:
I remember that someone ran arktos-up using his macbook and got some network interface issues. It will be ok if we don't support macbook.

Reply:
Sonya said she was never able to use macbook to build. Vinay's test also runs on Ubuntu only. Arktos is committed to support macbook.